apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -42,6 +45,11 @@ type ImagesSpec struct {
 	//
 	// +optional
 	TempoGatewayOpa string `json:"tempoGatewayOpa,omitempty"`
+
+	// OauthProxy defines the oauth proxy image used to protect the jaegerUI on single tenant.
+	//
+	// +optional
+	OauthProxy string `json:"oauthProxy,omitempty"`
```

Comment:
We were also thinking about using kube-rbac-proxy? Is there any cons/pros for using one or the other?

Reply:
The thing with kube-rbac-proxy is that is only for token based authentication. It doesn't support the oauth flow we need to protect UI stuff.